container/webui/docker-compose.yaml

```diff
@@ -79,6 +79,25 @@ services:
     deploy:
       replicas: ${OPENQA_WEBUI_REPLICAS}
 
+  worker:
+    image: openqa_worker
+    build: ../worker
+    privileged: true
+    devices:
+      - /dev/kvm
+    volumes:
+      - ./workdir/data/factory/iso:/data/factory/iso:rw
+      - ./workdir/data/factory/hdd:/data/factory/hdd:rw
+      - ./workdir/data/factory/other:/data/factory/other:rw
+      - ./workdir/data/factory/tmp:/data/factory/tmp:rw
+      - ./workdir/data/testresults:/data/testresults:rw
+      - ./workdir/data/tests:/data/tests:ro
+      - ./conf:/data/conf:ro
+    depends_on:
+      - nginx
+    deploy:
+      replicas: ${OPENQA_WORKER_REPLICAS}
```

Comment:
Would it make sense to have a `healthcheck` here? Something to see if the container came up 🤔

Reply:
Done

Reply:
I'll wait for #3773 since it's needed for the *healthcheck* to be run in CI. Please remind me to re-review once this PR has the check running